docs/Researcher/workloads/inference-overview.md

```diff
@@ -50,34 +51,36 @@ This has the benefit of conserving resources at the risk of a delay from "cold s
+During the update process of an inference workload, a new revision of pod(s) is created. This revision is the new desired specification of the workload. Although several updates can be submitted consecutively even if the process of the previous update is not complete, the target goal is always according to the last submitted update. This means, the previous updates are ignored.
 
-Once the new revision is created completely (according to the desired spec) and up and running, the entire traffic of requests is navigated to the new revision, and the original workload is terminated. Then the update process is considered complete.
+Once the new revision is created completely and up and running, the entire traffic of requests is navigated to the new revision, and the original workload is terminated. Then the update process is considered complete.
 
 It is important to note that:
 
-* To finish the inference workload update successfully, the project must have sufficient free GPU quota in favor of the update.  
-  For example:  
-    * Before the update: 3 replicas A running inference workload with 3 replicas (let's assume that each replica is equal to 1 GPU). This means the project is already using 3 GPUs of its quota. For the sake of simplicity, we will refer to this revision as revision #1.
+* To finish the inference workload update successfully, the project must have sufficient free GPU quota in favor of the update. For example:  
 
-    * The update: 8 replicas This means, to complete the update, an additional 8 GPUs of free quota is needed. Only when the update is complete, the 3 GPUs used for the 1st revision are reclaimed.
+    * The existing workload uses 3 replicas: A running inference workload with 3 replicas, assuming that each replica is equal to 1 GPU, means the project is already using 3 GPUs of its quota. For the sake of simplicity, we will refer to this revision as revision #1.
 
-* The Workload grid in the user interface always displays the configuration of the desired specification (the latest submitted update). The status of the workload still represents the service status. For example, per the example described in point 1, during the update, the status of the workload is still “running” as the service is still being provided to the consumers (using revision #1).
+    * The workload is updated to use 8 replicas: This means, to complete the update, an additional 8 GPUs of free quota is needed. Only when the update is complete, the 3 GPUs used for the initial running inference workload are reclaimed.
+
+* In the UI, the **Workloads table** displays the configuration of the latest submitted update. For example, if you change the number of replicas, the **running/requested pods** column will display the updated replicas requested. 
```

Comment:
instead of mentioning the number of replicas in this example let's talk about changing the image. so when the user changes to image #2 this will be presented in the image column.

the reason I am saying this is because changeing the number of replicas is more of a complex topic I rather not get into here.

Reply:
@liohill is the column called "image"?